.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -19,6 +19,7 @@
 - [ ] The PR has an appropriate "CHANGELOG: ..." label (If you are unsure of which label to add, ask the reviewers for guidance).
 - [ ] The PR follows the [style guide](https://github.com/oppia/oppia/wiki/Coding-style-guide).
 - [ ] The PR addresses the points mentioned in the codeowner checks for the files/folders changed. (See the [codeowner's wiki page](https://github.com/oppia/oppia/wiki/Oppia%27s-code-owners-and-checks-to-be-carried-out-by-developers).)
-- [ ] The PR is assigned to an appropriate reviewer.
+- [ ] The PR is **assigned** to an appropriate reviewer.
   - If you're a new contributor, please ask on [Gitter](https://gitter.im/oppia/oppia-chat) for someone to assign a reviewer.
   - If you're not sure who the appropriate reviewer is, please assign to the issue's "owner" -- see the "talk-to" label on the issue.
+  - Do not only request the review but also add the reviewer as an assignee.
```

Comment:
This should be a continuation of the second bullet, right? New contributors can't assign.

Reply:
Yes, done.